CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_quickbooks v0.17.0
```

Comment:
Aren't we planning on cutting this as a pre-release first? If so, let's document this as v0.17.0-a1. We will then need to make the same update in the README (we can leave the dbt_project.ymls as they are)

Reply:
updated